docs/guide.mdx

```diff
@@ -0,0 +1,419 @@
+---
+sidebar_position: 10
+sidebar_label: "Walkthrough"
+id: deployment-guide
+title: Deployment Guide
+description: Deployment Guide
+slug: /walkthrough
+---
+
+This guide provides detailed instructions for deploying a Kwil network and for interacting with the nodes in the network.
+
+## Deploy a Standalone Kwil Node
+
+### Configuration
+
+Initialize the root directory with the required configuration files by running the below command.
+
+```bash
+$ kwil-admin setup init -o ~/.kwild
+Successfully initialized node directory: ~/.kwild
+```
+
+This generates the new node's private key, genesis and config files in the specified root directory.
+If the `-o` flag is not used, the default location is `./.testnet`.
+
+```bash
+$ tree ~/.kwild
+~/.kwild
+├── abci
+│   ├── config
+│   │   └── genesis.json
+│   └── data
+├── config.toml
+└── private_key
+
+4 directories, 3 files
+```
+
+The configuration is initially set to its default values. These can be updated manually or overridden using command line flags or environment variables. Refer to the [configuration guide](./daemon/kwild#configuration-override) for more information on this.
+
+Notable settings in `config.toml` and `genesis.json` are described below.
+
+#### Application Configuration (`config.toml`)
+
+This file contains operational parameters for the blockchain and databases. Some vital configurations include:
+
+Application Configuration:
+- `app.grpc_listen_addr`: The address on which the user gRPC server listens (default is `localhost:50051`).
+- `app.http_listen_addr`: The address on which the HTTP gateway to the gRPC service listens (default `localhost:8080`).
+
+:::caution
+The current Kwil release does not require gas to execute transactions.
+Care should be taken not to expose the gRPC service without appropriate access controls.
+:::
+
+Set `app.rpctls=true` to enable TLS on the public gRPC server. If the server's TLS certificate is self-signed rather than issued by a root certificate authority, it must be provided to clients.
+
+Configure `app.sqlite_file_path` to specify the location of the SQLite database files. By default, these files are written to `<root_dir>/data/kwild.db`
+
+Blockchain Configuration:
+- `chain.rpc.listen_addr`: The address on which the consensus engine's RPC server listens (default is `localhost:26657`). For advanced node operator use only.
```

Comment:
This is cometbft's own RPC interface rather than kwild's.  I was uncertain if we even wanted to expose this at all.

Side note, it looks defined incorrectly in the sample toml configs as `laddr` still.  Same for the p2p address it seems.